pom.xml

```diff
@@ -49,26 +49,26 @@
     central -->
   <developers>
     <developer>
-      <id>ipaas</id>
-      <name>iPaaS Development Team</name>
+      <id>syndesis</id>
+      <name>Syndesis Development Team</name>
       <organization>redhat</organization>
       <organizationUrl>http://redhat.com/</organizationUrl>
     </developer>
   </developers>
 
   <scm>
-    <connection>scm:git:git@github.com:redhat-ipaas/connectors.git</connection>
-    <developerConnection>scm:git:git@github.com:redhat-ipaas/connectors.git
+    <connection>scm:git:git@github.com:syndesisio/connectors.git</connection>
+    <developerConnection>scm:git:git@github.com:syndesisio/connectors.git
     </developerConnection>
-    <url>http://github.com/redhat-ipaas/connectors/</url>
+    <url>http://github.com/syndesisio/connectors/</url>
     <tag>${project.version}</tag>
   </scm>
 
   <distributionManagement>
     <repository>
-      <id>bintray-redhat-ipaas-maven</id>
-      <name>redhat-ipaas-maven</name>
-      <url>https://api.bintray.com/maven/redhat-ipaas/maven/connectors/;publish=1</url>
+      <id>bintray-syndesis-maven</id>
+      <name>syndesis-maven</name>
+      <url>https://api.bintray.com/maven/syndesisio/maven/connectors/;publish=1</url>
```

Comment:
We don't need these any more - we can distribute straight to Maven Central now. @rhuss Did you have the details for io.syndesis gid?